legend-engine-xt-persistence-test-runner/src/main/java/org/finos/legend/engine/testable/persistence/assertion/PersistenceTestAssertionEvaluator.java

```diff
@@ -40,20 +42,58 @@ public class PersistenceTestAssertionEvaluator implements TestAssertionEvaluator
 {
     private List<Map<String, Object>> result;
     private Set<String> fieldsToIgnore;
+    private Map<String, Object> milestoningMap;
 
-    public PersistenceTestAssertionEvaluator(List<Map<String, Object>> result, Set<String> fieldsToIgnore)
+    public PersistenceTestAssertionEvaluator(List<Map<String, Object>> result, Set<String> fieldsToIgnore, Map<String, Object> milestoningMap)
```

Comment:
We don't need a Map here. (milestoningMap)
All we need is one column name and one value. Rather than map, we can create a class containing the column and value.
Also calling it by name milestoningMap can a bit misleading. We can call it ActiveRowsFilterCondition.
If you agree, we can make changes in all the place related to this!

Reply:
Sure, changed!